helical/models/geneformer/geneformer_utils.py

```diff
@@ -36,6 +37,12 @@ def load_mappings(gene_symbols):
     pkl.dump(gene_id_to_ensemble, open('./human_gene_to_ensemble_id.pkl', 'wb'))
     return gene_id_to_ensemble
 
+def forw(model, input_data_minibatch, minibatch):
```

Comment:
seems like a residual function from testing?

Reply:
OK no sorry my bad. Might be worth calling it differently (or directly implementing it in the model file and not here)

Reply:
It has been removed with the changes for bringing the fine tuning train function out of the utils folder